src/Header.js

```diff
@@ -29,6 +34,15 @@ const Header = () => {
           <Dropdown.Item href="/aquarius">Aquarius</Dropdown.Item>
           <Dropdown.Item href="/pisces">Pisces</Dropdown.Item>
         </DropdownButton>
+        <DropdownButton
+          className="drop-button"
+          id="dropdown-basic-button"
+          title="HOROSCOPE"
+        >
+          <Dropdown.Item href="/yesterday">YESTERDAY</Dropdown.Item>
```

Comment:
I would like to add this dropdown as an additional feature.  I am not quite sure how to do this yet, so I will work on this next.  The dropdown is styled, just not functional.